jack-test/test/java/com/rapleaf/jack/transaction/TestDbTransactorImpl.java

```diff
@@ -148,6 +148,7 @@ public void testSimultaneousQuery() throws Exception {
 
     int actualCount = transactor.query(db -> db.createQuery().from(User.TBL).fetch().size());
     assertEquals(createCount1 + createCount2, actualCount);
+    transactor.close();
```

Comment:
There are many transactors in this test file, and only three are closed. Either close them all, or leave the test file as is. The `close` method is tested in dedicated test case, and this change does not affect the close function. So I think we should not change this file.